subsystems/intakesubsystem.py

```diff
@@ -106,7 +101,12 @@ def centerNote(self, frontLimitState, backLimitState) -> None:
                 NEOBrushless.ControlMode.Percent, constants.kIntakePercentageVoltage
             )
 
-    def holdingState(self, backLimitState: bool) -> None:
+    def holdingState(self, frontLimitState: bool, backLimitState: bool) -> None:
+        if self.putInPlace:
+            pass
+        else:
+            self.centerNote(frontLimitState, backLimitState)
+            self.setPivotAngle(constants.kStagingPositionAngle)
         self.setPivotAngle(constants.kHandoffAngle)
```

Comment:
this and the other stuff will happen regardless of putInPlace, move what you want to happen inside the if statement